superset/superset/assets/src/visualizations/presets/MainPreset.js

```diff
@@ -85,8 +84,6 @@ export default class MainPreset extends Preset {
         new LineChartPlugin().configure({ key: 'line' }),
         new LineMultiChartPlugin().configure({ key: 'line_multi' }),
         new MapBoxChartPlugin().configure({ key: 'mapbox' }),
-        new MarkupChartPlugin().configure({ key: 'markup' }),
-        new MarkupChartPlugin().configure({ key: 'separator' }),
```

Comment:
oh, they were really exactly the very same thing?

Reply:
the difference is here: https://github.com/apache-superset/superset-ui-plugins/blob/master/packages/superset-ui-legacy-plugin-chart-markup/src/Markup.jsx#L52

Reply:
thanks for poiting it out!